uiowa/uiowa
# Feature scholarship weights

Resolves #2436; followup issues may need to be added for:
 - As an Admissions site owner, I can order scholarships content in relation to each other (e.g. see weights for other content, drag and drop, or other option) so that prioritizing scholarships is faster and has less room for human error.
 - As a site owner, I can weight content which appears in list views in ways other than automatic sorting, so that I can prioritize displays for criteria which is not based on data within the system.

# How to test

`git pull`
`composer install`
Sync admissions, run the update hook if working off a previously synced version
Add a Scholarships block to a page, note the default ordering (alphabetical by title)
Update weights for various scholarships (both positive and negative need testing)
Check Scholarships block, see that the ordering is correctly updating

Check that sites not on the admissions split are unaffected.
